hello-world/hello_world_test.rb

```diff
@@ -12,33 +12,25 @@
   exit 1
 end
 
+# Test data version:
+# 7668b09 Added hello world test definition
+
 class HelloWorldTest < Minitest::Test
   def test_no_name
-    assert_equal 'Hello, World!', HelloWorld.hello, <<-MSG
-    When given no name, it should greet the world!
-    MSG
+    assert_equal 'Hello, World!', HelloWorld.hello,
+                 'When given no name, it should greet the world!'
   end
```

Comment:
So here's an interesting discussion, and I stopped creating automated tests on this one as this is a common problem that is going to occur repeatedly.

The problem is that for most of the tests where there are messages present, the lines are longer than 80 characters.

My choices were:
 -Use HEREDOC, which appears to be the ideal and recommended thing to use for multi-line test literals. This does introduce quite a lot of noise and density into the code, as it changes every test into three lines, and indents the test message against the right, which does not read well. 
- Split the line and indent the message. While it does affect readability, it keeps the code light, and I believe easier to grok than the the HEREDOC approach.
- Assign the message to a variable and then use the variable. I really believe that this is a box-ticking exercise that adds complexity into the code, and obscures easy reading. 
- Attempt to squish the messages which sort of defeats the purpose of having the messages at all.

It feels like the only reason to do any of the three is to tick the rubocop box, as none adds either readability nor functionality. They all feel like a compromise:(

When I looked at the code generated for something long like the RNA problem, it became clear (to me anyway), that the HEREDOC and variable approaches made the code denser, longer, and not more easy the read or understand which is the key driver for the theory behind style.

I do feel somewhat like a lawyer making a case, but this affects much of the work to be done for all of the test generation when messages come into play, so it's worth a bit of effort.

Thoughts?